src/LSP3/getProfileMetadata/getProfileMetadata.ts

```diff
@@ -67,9 +67,12 @@ export async function getProfileMetadata(
 
     const profileDataURL = validateIpfsUrl(JSONURL.url);
 
-    const profileData: LSP3ProfileMetadata = await fetch(profileDataURL).then((response) =>
-        response.json(),
-    );
+    let profileData: LSP3ProfileMetadata;
+    try {
+        profileData = await fetch(profileDataURL).then((response) => response.json());
```

Comment:
Don't combine await and then

User either only await or only then.

```suggestion
        const response = await fetch(profileDataURL);
        profileData = await response.json();
```

https://stackoverflow.com/a/69347918

Reply:
I see, makes sense, thanks for tip 👍